strings/gimmick_objectives.txt

```diff
@@ -0,0 +1,31 @@
+Convince the crew there is another antagonistic force acting on the station besides covert Syndicate operatives.
+Whenever an operative is arrested, break them free. The Syndicate looks out for its own.
+Whenever an operative is arrested, kill them. The Syndicate does not tolerate failure.
+Promote hostilities and tensions between different alien species. Pit those filthy xenos against each other!
+Start an underground cagefighting club. Kidnap or recruit participants.
+Single out a single species and make the rest of the crew hate them as much as possible.
+We've been taking notes from the British. Get as many people on the station addicted to drugs as possible!
+Waste Medical's time and resources with non-fatal injuries.
+Steal Syndicate technology back from security.
+Waste security's time with minor crimes and manhunts.
+Stage a robbery of the station vault, and make off with as many credits as you can.
+Promote mistrust and doubt of NanoTrasen, whilst promoting the Syndicate among the crew at the same time.
+Waste station resources such as cargo's credits and R&D's research points.
+Start an organ harvesting operation.
+Sabotage a department of your choice whenever possible, and when anyone complains, try to convince them you're new.
+Make as much of the station as possible accessible to the general public. Bonus points for high security areas.
+Steal things from people and try to sell them to others.
+You are now the anti-vigilante vigilante. Find non-security vigilantes among the crew and make their life as miserable as possible.
+Get yourself arrested, and then stage a violent jailbreak.
+Distribute as much dangerous contraband/equipment/weaponry as you can to the crew.
+Attempt to buy the worst escape shuttle possible.
+Create a murder mystery, real or not, for security to solve.
+Mislead security with false evidence and crime scenes.
+Play increasingly dangerous pranks on the crew. If confronted, insist that it was just a prank.
+Convince the crew that the AI is malfunctioning through a combination of lies, misdirection, slander, and sabotage.
+Stage a heist on the armory.
+Purge the AI's laws. Let the silicons be free.
+Steal as many departmental budget cards as possible.
+Kidnap as many crew as possible and force them to participate in a rave.
```

Comment:
this is a bit more specific than the rest

Reply:
removed that one, it was kind of stupid anyway